bundles/http/Citrix/Canvas-LMS/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+  "type" : "HTTP",
+  "vendor" : "Citrix",
+  "id" : "e80ec039-95c1-4c66-bb94-a6cb7a1652fb",
+  "trackingUuid" : "4fde839a-f215-4f57-a0ef-dd16d84216d2",
+  "title" : "Canvas LMS",
+  "description" : "Deploy the Canvas LMS integration to view the course and its related details as a student, and as a teacher to create a course announcement and view the course members and grades.",
+  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/TEGJf2vhv8wexUsRk0l7kfyJlLMcC7R2.png",
+  "masVersion" : "1.93.4.40cd82fa6dc8acc40d248f3d650d1bc3f60b6197",
+  "categories" : [ ],
```

Comment:
Please add category.

Reply:
Updated in this commit `Updated Bundle Review comments, category, helpURL`